README

```diff
@@ -19,11 +19,10 @@
 
     ~ What do I need?
 
-      Jinja 2.4 and Werkzeug 0.7 or later.
-      `pip` or `easy_install` will install them for you if you do
-      `pip install Flask`.  I encourage you to use a virtualenv.
-      Check the docs for complete installation and usage
-      instructions.
+      If you do `pip install Flask`, `pip` or `easy_install` will
```

Comment:
That sentence is a bit warped (has always been), executing `pip` doesn't somehow launch `easy_install`.